composer/modules/web/src/plugins/import-struct/dialogs/import-struct-dialog.jsx

```diff
@@ -56,7 +56,8 @@ class ImportStructDialog extends React.Component {
      * Called when user clicks 'Import Struct' menu item.
      */
     onImportJson() {
-        if (this.props.onImport(this.state.json)) {
+        const { structName } = this.state;
```

Comment:
@DimalChandrasiri You can directly pass this.state.structName as its been done for this.state.json. I think better to follow one approach for both params

Reply:
fixed with commit 0974ece26d1cb2557d0f080e28c729a8122f138d